forta-agents-linked/src/agent.js

```diff
@@ -18,11 +18,17 @@ function provideHandleTransaction(timeHandler) {
   return async function handleTransaction(txEvent) {
     const findings = [];
     const filtered = txEvent.filterLog(eventABI, contractAddress);
+    let aboveThreshold = [];
     filtered.forEach((tx) => {
-      timeHandler.addToList(tx.args.agentId);
+      const { name } = tx;
+      if (name == "AgentUpdated") {
```

Comment:
i think we'll need to check for creation only here (not updates). when a bot is updated, the assigned scan node does not change